### PR TITLE
Add macro for Sauce Labs promos. Bug 1330094.

### PR DESCRIPTION
**Pull request by jpetto**
Macro to display one of three Sauce Labs promos.

### CODE REVIEW

**Comment by jpetto** [comment]
Depends on [kuma PR 4139](https://github.com/mozilla/kuma/pull/4139).

File: macros/SauceLabsPromo.ejs [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<%
+var version = (arguments.length) ? $0 : 'a';
+
+if (version === 'a') { %>
+  <% // runs on /docs/Tools %>
+  <div class="wiki-promo-sauce-labs" id="wiki-promo-sauce-labs-a">
+    <div class="wiki-promo-copy">
+      <p><strong>Learn about manual, automated and parallel testing with our Cross-browser Testing Tutorial.</strong></p>
```
Comment (schalkneethling) [inline macros/SauceLabsPromo.ejs]: Guessing these strings are not being localised as they are here? https://github.com/mozilla/kuma/pull/4139/files#diff-141fd90a29db7f8735f7d16f5c45dd76R128

Comment (jpetto) [inline macros/SauceLabsPromo.ejs]: Yes, I was told we are not localizing this promo.

**Comment by jpetto** [comment]
**DO NOT MERGE**

Looks like [copy is going to change](https://bugzilla.mozilla.org/show_bug.cgi?id=1330094#c30). Shouldn't affect code review though.

**Comment by jpetto** [comment]
Copy is updated and the kuma PR is merged. This one is now ready.

**Pull request merged**